internal/batch/batch.go

```diff
@@ -345,7 +348,11 @@ func (s *scheduler) build(ctx context.Context, pkg string) error {
 		return err
 	}
 	defer logFile.Close()
-	build := exec.CommandContext(ctx, "distri", "build")
+	archOpt := ""
+	if s.arch != "" {
+		archOpt = fmt.Sprintf("-cross=%s", s.arch)
+	}
+	build := exec.CommandContext(ctx, "distri", "build", archOpt)
```

Comment:
You can write that like so to make it a little easier to follow:

```go
build := exec.CommandContext(ctx, "distri", "build")
if s.arch != "" {
  build.Args = append(build.Args, "-cross="+s.arch)
}
```

Reply:
Huh, didn't know that. My go is a bit patchy in various places...